### PR TITLE
Add a README

### LINKED ISSUE #116: Add a readme

### PR DESCRIPTION
**Pull request by bhazelton**
closes #116

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,68 @@
...
+Please submit improvements as pull requests against the repo after verifying that
+the existing tests pass and any new code is well covered by unit tests.
+
+Bug reports or feature requests are also very welcome, please add them to the
+issue log after verifying that the issue does not already exist.
+Comments on existing issues are also welcome.
+
+# Versioning Approach
+We use a `generation.major.minor` format.
+* generation - certified comparisons with other simulations
```
Comment (jpober) [inline README.md]: I would strongly prefer something like "Major algorithmic modifications or substantive new features added.  Always accompanied by in-depth comparisons with other simulations."

Comment (dannyjacobs) [reply README.md]: I'm rethinking this a little bit after talking with Steven and Adam here.   Steven says that when he sees a generation version change he expects a significant change but in major versions new things are "optional". They both agreed that the use of the major version to distinguish "level of validation" while sensible from our perspective, might be excessively novel and perhaps a little opaque for others.  And, on reflection I see that we've under-defined the major category.  We also explored what "algorithmic" meant to us and discovered that to some it implied "physics" and to others "computation". In reality of course they are somewhat correlated. Heres a suggested version where I've stuck to those two words. I've also explicitly enumerated the testing and validation strategy for each version.

Generation - Release combining multiple new physical effects and or major computational improvements. Testing: Backed by unittests, internal model validation,  and significant external comparison.
Major - Adds new physical effect or major computational improvement. Small number of improvements with each release. Testing: Backed by unittests, internal model validation and limited external comparison.
Minor - Bug fixes and small improvements not expected to change physical model. Testing: Backed by Unittests

Comment (aelanman) [reply README.md]: Sounds good to me!
Just to clarify a little for my own understanding... and to give examples of things we've discussed doing:
 - "physical effects"  = things like polarization effects, noise, ionospheric modeling, or nonterrestrial observing positions.
 - "Major computational improvement" = Support for new catalog types (e.g, diffuse maps), new analysis tools, changes to parallelization scheme
 - "Small improvements" = Better documentation or example code, outer framework redesign.

Is this about right?

Comment (jpober) [reply README.md]: This sounds like a good plan to me.  If you can push the changes, I'll finish the review.

**Pull request merged**